crates/models/src/tables/macros.rs

```diff
@@ -318,13 +348,15 @@ macro_rules! tables {
                 let mut stmt = db.prepare(&Self::select_sql())?;
                 self.0.extend(stmt.query_map([], $row::scan)?
                               .collect::<Result<Vec<_>, _>>()?);
+                self.reindex();
```

Comment:
I think you can replace `self.0.extend` + `self.reindex` with `self.extend` now, right?

Reply:
Done.